src/components/AppContentListItem/AppContentListItem.vue

```diff
@@ -0,0 +1,88 @@
+<!--
+  - @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<a
+		href="#"
+		class="app-content-list-item">
+		<!-- default slot for avatar or icon -->
+		<slot />
+		<div
+			class="app-content-list-item-line-one">
+			{{ title }}
+		</div>
+		<div
+			class="app-content-list-item-line-two">
+			{{ subtitle }}
```

Comment:
what if there is none? Should we make the line-two optional?

Reply:
IMO it should be a slot. Otherwise Mail can't use it https://github.com/nextcloud/mail/blob/c1ef17b8b631bf14ef5a1bc61f2e330f45907b03/src/components/Envelope.vue#L19-L26

Reply:
> IMO it should be a slot. Otherwise Mail can't use it

Wait, are you talking about title and subtitle? Can't you do `:title="addresses"` and `:subtitle="data.subject"`? For the icon and actions there's indeed a slot.

Reply:
Yes and no. The subtitle, for example, is italic for unread messages. How can I achieve that with a string prop?

Reply:
@ChristophWurst :drum: :drum:  CSS :grin:

Reply:
Got it! but how many exceptions of this kind do we have across these apps? If it's just a few of them they could be just more props.

Reply:
Or a slot :wink:

Reply:
A slot would be the easiest way  @ma12-co :)
Allowing text and whatever in it :)

Reply:
>Allowing text and whatever in it :)

But isn't this supposed to standardize things across the apps? Why would we want to allow anything other than text in title and subtitle fields???

Reply:
Because there is a fine line between too much restriction and too much freedom :)
I'd say it's fine to allow a few icons here! Let's not recommend it and make sure only one line is displayed !